### DISCUSSION TITLE
Server Side Calls with `t.createCallerFactory`

### DISCUSSION THREAD

**Discussion by zirkelc** [trpc/trpc#6159] [Q&A] [answered]
Hi,

in v10 it was not recommended to use `t. createCaller` to do [server side calls](https://trpc.io/docs/server/server-side-calls). In v11 (or maybe even v10) a new function `t.createCallerFactory` was introduced. Since the warning in the docs still refers to `t. createCaller`, I wanted to ask if it is still valid not to use `t.createCallerFactory` for server side calls?

**Comment by KATT** [comment, marked as answer]
Yeah, you can use `.createCaller()`.... might actually remove the deprecation note eventually
